packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -296,6 +322,11 @@ export default function ProjectNav({
   const isCloudManager = current?.planType === PlanType.Cloud;
   const isLoading = !!current;
   const currentProjectId = current?.projectId;
+  const useCNRegionsOnly =
```

Comment:
could we do something like the following to avoid the duplicated product logic below
```ts
const useCNRegionsOnly = current?.useCNRegionsOnly ?? false;
const planType = current?.planType;

const isCNRegionsOnly = useCallback(
  (product: Product) => {
    const disabledProducts: Array<Product> = [Product.Realm, Product.Charts];

    return (
      useCNRegionsOnly &&
      planType === PlanType.Atlas &&
      disabledProducts.includes(product)
    );
  },
  [useCNRegionsOnly, planType],
);
```

Reply:
👍